apps.yml

```diff
@@ -4018,3 +4018,15 @@ apps:
     url: https://auth.mozilla.auth0.com/samlp/FoeZGnqq0X9fQXQQ9Ve5kdSdPN4eedlG
     vanity_url:
     - /prefect-pocket
+- application:
+    authorized_groups:
+    - team_moco_benefited
```

Comment:
Does mofo expect to have access to this as 'benefited employees' in general? If so, team_mofo, I believe, is all that's required (they don't differentiate benefited beyond that).

Reply:
I'll double check on whether mofo should be included

Reply:
MoFo will not be covered.  It’s a completely different legal entity and they offer their own set of benefits.